scripts/schemas.js

```diff
@@ -55,4 +55,5 @@ const main = async () => {
   }
 }
 
+// this is a main function
```

Comment:
You can remove the comment now, and you won't need the devops reviewers anymore as well

Reply:
NOOO the build failed again 🙃